core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/ddl.scala

```diff
@@ -59,6 +59,7 @@ case class StructField(name: String, dataType: DataType, nullable: Boolean = tru
 case class StructType(fields: Seq[StructField]) extends DataType
 case class MapType(keyType: DataType, valueType: DataType) extends DataType
 case object VariantType extends DataType
+case class ColumnDetail(name: String, dataType: DataType, nullable: Boolean = true, comment: Option[String] = None)
```

Comment:
Can you add a comment on what or why this IR is needed

Reply:
As we can't use StructField to store the comments for each column